### ISSUE TITLE
Automatic removal of dependencies

### ISSUE THREAD

**Issue by anders-kiaer** [equinor/komodo#70]
When removing a package from a release file, it would be nice to also at the same time automatically be able to remove other packages being dependencies (direct + indirect) which are not used by other packages in the same release dependency tree.

**Comment by sondreso** [comment]
Thanks for the feature suggestion! We are in the process of adding quite a bit of tooling to komodo to easy the deployment process, and this feature is one of the things that (in some form) will be implemented :slightly_smiling_face:

**Comment by oysteoh** [comment]
@mortalisk I guess your tool would be easy to use to compile a list at least of dependencies not in use?

**Comment by mortalisk** [comment]
Yes, since it finds reverse dependencies, detecting when there are no dependencies should be relatively easy. I have a pull request here: https://github.com/equinor/komodo/pull/141

**Comment by jondequinor** [comment]
I would strongly recommend https://github.com/equinor/komodo/issues/148 to be solved before this one.

**Comment by eivindjahren** [comment]
There is now tools that automatically detect unused packages.
